### PR TITLE
Add more cpu and mem to multipass vm.

### PR DESCRIPTION
**Pull request by jasondoze**
Signed-off-by: Jason Doze <dozejason@gmail.com>

### CODE REVIEW

File: crown-apparel/install.sh [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+if ( which node ) 
+then
+  echo -e "\n==== Node already installed ===="
+else 
+  echo -e "\n==== Installing Nodejs && npm ===="
+  sudo apt install -y nodejs
+fi
+
+# cd into the crown app directory
+if [[ ${PWD} = */crown-apparel/* ]]
```
Comment (stephenlauck) [inline crown-apparel/install.sh]: In the deploy you are just using a `cd` to the right directory, do we need to check again here? What is the scope of concern here and is this reaching to far out of it from the install.sh perspective?

File: crown-apparel/install.sh [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+echo -e "\n==== Install complete ====\n"
+
+echo "                                                                                                         "
+echo "  _________                                       _____                                           .__    "
+echo "  \_   ___ \ _______   ____  __  _  __  ____     /  _  \  ______  ______  _____   _______   ____  |  |   "
+echo "  /    \  \/ \_  __ \ /  _ \ \ \/ \/ / /    \   /  /_\  \ \____ \ \____ \ \__  \  \_  __ \_/ __ \ |  |   "
+echo "  \     \____ |  | \/(  <_> ) \     / |   |  \ /    |    \|  |_> >|  |_> > / __ \_ |  | \/\  ___/ |  |__ "
+echo "   \______  / |__|    \____/   \/\_/  |___|  / \____|__  /|   __/ |   __/ (____  / |__|    \___  >|____/ "
+echo "          \/                               \/          \/ |__|    |__|         \/              \/        "
```
Comment (stephenlauck) [inline crown-apparel/install.sh:47]: :metal:

File: crown-apparel/install.sh [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+# make sure node setup is installed
+
+
+
+
+
+
+
+<<nmap
```
Comment (stephenlauck) [inline crown-apparel/install.sh]: What are you trying to do from here down? Can you just write out a comment that is like a test or assertion of what you think you need instead? You looking for the IP address?

File: crown-apparel/install.sh (hunk 1 above)
Comment (jasondoze) [reply crown-apparel/install.sh]: Yep, thats right. I removed it. Maybe confused ~/crownapp with ~/crown-apparrel

File: crown-apparel/install.sh (hunk 3 above)
Comment (jasondoze) [reply crown-apparel/install.sh]: Here I was testing nmap, lsof, and ss. Looking for ways to implement these:
# show that the app is running through the ip address
# and curl ip
Moved to notes.

**Pull request merged**